README.md

```diff
@@ -134,6 +134,96 @@ class ArtistFilter(AutocompleteFilter):
         return reverse('admin:custom_search')
 ```
 
+Shortcut for creating filters:
+------------------------------
+
+It is now possible to use the `ACFilter` shortcut to create filters
```

Comment:
Could be `AutocompleteFilter` for clarity?

Reply:
Agreed with @merwok

Reply:
Yeah, the short name was not a great idea. But there's already an `AutocompleteFilter` in that module (the filter class itself), so I honestly wasn't quite sure what to name that function. I tend to think that they should have different names for clarity, not just be separated. Do you have a suggestion?

Reply:
Is it possible to merge the two classes?

The existing AutocompleteFilter requires subclassing, your addition enables a nicer style, the `__init__` function could be made to support both cases?

In my own projects, I have tried to write filter classes supporting this style (from django docs):
```python
class PersonAdmin(admin.ModelAdmin):
    list_filter = (
        ('is_staff', admin.BooleanFieldListFilter),
    )
```
but ended up making a factory function that returned custom subclasses of SimpleListFilter.

Reply:
In my additional commit, I renamed `ACFilter` to `AutocompleteFilterFactory`. It's not really that I have another class, it's that I just want provide a shortcut for generating a customized `AutocompleteFilter` class directly inside `list_filter`, rather than defining a new filter class manually. In this approach I can't really combine classes - there's really still only the one class (just with a couple methods overridden).

As for the custom format, would you strongly prefer the syntax for the second filter here, rather than the first?

```python
class PersonAdmin(admin.ModelAdmin):
    list_filter = (
        AutocompleteFilterFactory('food (auto)', 'favorite_food', ...),
        ('favorite_food', AutocompleteFilterFactory('food (auto)', ...)),
    )
```

I think I can make the second form work (potentially both), but I went with the first form initially because it was clearer to me. I was thinking about the form (also from the docs) that looks like `list_filter = (DecadeBornListFilter,)`, just with a customized class.

Let me know what you think - happy to modify.